src/components/ConfirmationPage.js

```diff
@@ -46,7 +46,7 @@ function ConfirmationPage(props) {
                     source={props.animation}
                     autoPlay
                     loop
-                    style={styles.confirmationAnimation}
+                    style={[styles.confirmationAnimation, styles.aspectRatioLottie(props.animation)]}
```

Comment:
We could possibly add some kind of wrapper to the `<Lottie>` component, adding this style, instead of copying the same line several times, with the argument, that is already passed to the component.
Just an idea.